coalib/output/ConsoleInteraction.py

```diff
@@ -697,6 +697,18 @@ def show_bear(bear,
                          metadata.optional_params,
                          "  ",
                          "No optional settings.")
+        show_enumeration(console_printer,
+                         "Can detect:",
+                         bear.can_detect,
+                         "  ",
+                         "This bear does not provide information about what "
```

Comment:
Do we want to say this for each bear. A little spammy when I try to print 14-15 bears. IMO not printing the whole section is better.